packages/hint-manifest-icons/tests/tests.ts

```diff
@@ -214,7 +214,123 @@ const tests: HintTest[] = [
                                 "sizes": "192x192",
                                 "type": "image/png"
                             }
-                            ]
+                        ]
+                    }`
+            }
+        }
+    },
+    {
+        name: 'Duplicated purpose found',
+        reports: [
+            {
+                message: `Duplicate value(s) found in icon purpose porperty: any.`,
+                position: {
```

Comment:
Maybe we could use the position matching so this is easier to maintain in the future?

Reply:
That would be awesome, but does it work with this match `icons[0].purpose` because I think I tried and it didn't work. (I am going take another look)

Reply:
I think I can use `purpose` for the match, but it will only match the first occurrence of `purpose` and there are 3 icons in the tests.

Reply:
I thought it was a bit more flexible. Not worth doing it then.

Reply:
The position matching should work here, we just need to include enough of the matching line to be unique within the file.